build/builders/index.md

```diff
@@ -0,0 +1,76 @@
+---
+title: "Builders"
+keywords: build, buildx, builders, buildkit, drivers, backend
+description: 
+---
+
+A builder is a BuildKit daemon that you can use to run your builds. BuildKit
+is the build engine that solves the build steps in a Dockerfile to produce a
+container image or other artifacts.
+
+You can create and manage builders, inspect them, and even connect to builders
+running remotely. You interact with builders using the Docker CLI.
+
+> **Note**
```

Comment:
Should we use the experimental callout like https://docs.docker.com/build/dockerfile/release-notes/#150-labs?

Reply:
Huh, didn't even know we had this. I quite like the look of this - makes it clear that it's different to a regular note.

Reply:
Added something. It doesn't exactly line up with the notes we've used elsewhere, but maybe it's a pattern that can catch on :-) cc @aevesdocker WDYT?

Reply:
Sorry, only just seen this notification! I like this pattern a lot. I'll add it to the contribute guide